esp/platform/application_config.cpp

```diff
@@ -107,29 +111,81 @@ bool addSecurity(IPropertyTree *legacyEsp, IPropertyTree *appEsp, StringBuffer &
 
     VStringBuffer authenticationXml("<Authentication htpasswdFile='/etc/HPCCSystems/.htpasswd' ldapAuthMethod='kerberos' ldapConnections='10' ldapServer='%s' method='ldaps' passwordExpirationWarningDays='10'/>", configname.str());
     legacyEsp->addPropTree("Authentication", createPTreeFromXMLString(authenticationXml));
+    return true;
+}
 
+bool addAuthNZSecurity(const char *name, IPropertyTree *legacyEsp, IPropertyTree *appEsp, StringBuffer &bindAuth)
+{
+    IPropertyTree *authNZ = appEsp->queryPropTree("authNZ");
+    if (!authNZ)
+        throw MakeStringException(-1, "can't find application AuthNZ section.  To run without security set auth=none");
+    authNZ = authNZ->queryPropTree(name);
+    if (!authNZ)
+        throw MakeStringException(-1, "can't find application %s AuthNZ settings.  To run without security set auth=none", name);
+    IPropertyTree *appSecMgr = authNZ->queryPropTree("SecurityManager");
+    if (!appSecMgr)
+        throw MakeStringException(-1, "can't find SecurityManager settings.  To run without security set auth=none");
```

Comment:
There is only one app instance per ESP when running in this mode.  The entire log file would be for that app.  I could grab the name from the config if you think it's good to add it to the line anyway?

Reply:
If this exception's message is only exposed through the log file dedicated to this one app, it might be superfluous...
Consider though, by default k8s/docker will be coalescing all log output to a single stream/file

Reply:
Ok, I haven't seen that in action yet.  I'll add the app name.  In my direct use of k8s I directly ask for a pod/container's log.